.github/CODEOWNERS

```diff
@@ -103,9 +103,8 @@
 /docs/coding-guidelines/typescript* @Automattic/type-review
 
 # FSE (Full Site Editing)
-/apps/full-site-editing/* @Automattic/cylon
-/apps/full-site-editing/full-site-editing-plugin/* @Automattic/cylon
-/apps/full-site-editing/full-site-editing-plugin/full-site-editing @Automattic/cylon
```

Comment:
This folder no longer exists. It has been renamed to `dotcom-fse` and this reference wasn't updated.